styles.css

```diff
@@ -0,0 +1,36 @@
+body {
+  text-align: center;
+  font-family: Helvetica, sans-serif;
+  font-size: 2.5rem;
+  color: darkblue;
+}
+
+.background {
+  background-image: url("http://www.tourneau.com/on/demandware.static/-/Sites-Tourneau-Library/default/dwe469400e/images/cpo/2.06.01_New_Background_CPOBL_Desktop_450.jpg");
+  background-size: 100% 100%;
+  width: 100%;
```

Comment:
`div` elements are `display: block` by default (100% width). So you can delete this `width` property. Check all of your CSS properties and make sure they are needed. Deleting code is a win!